src/projects/EnsureThat/ExceptionMessages.cs

```diff
@@ -8,6 +8,7 @@ public class ExceptionMessages
         public static string Booleans_IsFalseFailed { get; } = "Expected an expression that evaluates to false.";
 
         public static string Collections_Any_Failed { get; } = "The predicate did not match any elements.";
+        public static string Collections_Contains_Failed { get; } = "The collection did not contain the expected item.";
```

Comment:
What does ContainsKey use for it's template filling? Does it enforce `string`, or does it .ToString(). If it `.ToString()`s, so should `Contains`